primes.rb

```diff
@@ -1,6 +1,30 @@
 class Primes
-  def self.sum_to(limit = 100)
+  def self.sum_to(limit)
     # TODO - add your prime number solution here...
     "I'm working on it!"
+    sum  = 0;
+    for  i in 2..limit
+       isNotPrime =  true
+    	for j in 2..i 
+    	
+           if i%j==0 && i!=j
+           	    isNotPrime = false
+         
+              end
+          end
+          if isNotPrime == true
+            sum= sum+ i
+          end
+      end
+      puts sum
+     return sum
```

Comment:
This is a good start. Try to get in the habit of using good indentation in your code, as this makes it easier to spot problems with syntax. Could you make your code more efficient by skipping over values that couldn't possibly be prime? What about numbers that divide by two for instance?